imaginairy/cli/upscale.py

```diff
@@ -40,27 +31,15 @@ def validate_ext(ctx, param, value):
     "--upscale-model",
     multiple=True,
     type=str,
-    default=["realesrgan"],
+    default=["ultrasharp"],
     show_default=True,
     help="Specify one or more upscale models to use.",
 )
 @click.option(
     "--format",
-    default="{original}_upscaled_{number}_{algorithm}_{architecture}_{date}_{time}.{ext}",
-    type=str,
-    help="Format file name.",
-)
-@click.option(
-    "--date",
-    callback=validate_date,
-    help="Specify the date format, used with --format.",
-    type=str,
-)
-@click.option(
-    "--ext",
-    callback=validate_ext,
-    help="Specify the file extension, used with --format.",
+    default="{original}_upscaled_{number}_{algorithm}_{date:%Y-%m-%d}_{time:%H-%M-%S}",
```

Comment:
variable names are too vague.  The first time a person reads this they should know exactly what they are.

Reply:
If you were upscaling a file, is this the format you'd find most useful?

Reply:
I've updated the variable names, joined date and time to be now, and updated the default structure to match something closer to how standard images are formatted.

Reply:
For a default setting I wouldn't want the date and time as I don't think it's that useful.

I like the format the imaginiary generates file names, so it is now something closer to that along with a "UPSCALED" word added to it.